ansible-test/my-playbook.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: local-test
+  remote_user: root
+  tasks:
+    - name: add some config the .bashrc
+      become: yes
+      blockinfile:
+        path: /home/qujiabao/.bashrc
+        block: |
+          export DOCKER_HOST="0.0.0.0:2376"
+
+    - name: copy the daemon.json to /etc/docker
+      become: yes
```

Comment:
You can `become: yes` at the top level so that you don't have to `become` for every task.